analysis_notebooks/Object Analysis.ipynb

```diff
@@ -1409,11 +1409,11 @@
     "    ax.tick_params(axis=\"y\", left=True, right=False, labelleft=True, labelright=False, labelrotation=0)\n",
     "    plt.ylabel('Scenes')\n",
     "    plt.xlabel('Quantity')\n",
-    "    plt.tight_layout()\n",
     "    plt.title(\"Distribution of scenes that appear in dataset\")\n",
+    "    plt.tight_layout()\n",
     "    if first_pass:\n",
     "        to_write[7] = [\"(obj_scn) Distribution of scenes that appear in dataset.\"]\n",
-    "        plt.savefig(\"results/{0}/{1}/obj_scn.png\".format(folder_name, save_loc))\n",
+    "        plt.savefig(\"results/{0}/{1}/9.png\".format(folder_name, save_loc), bbox_inches='tight')\n",
```

Comment:
is this supposed to be 9.png or obj_scn.png? I'm actually not sure

Reply:
When the summary pdf gets written it looks for '9.png' and this appears to be the image it's looking for, otherwise I'd change the image name in the summary pdf generation cell but it wouldn't be consistent with the other images.